### PR TITLE
Release 1.4.5: update changelog wording

### PR DESCRIPTION
**Pull request by KievDevel**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -17,10 +17,10 @@
 
 -   Blocking dialog when disconnecting a device when the Programming dialog is
     open.
--   No longer auto reselect to the least disconnected device if is is in the
-    device list when clicking the `Auto Reconnect` Toggle.
+-   No longer auto reselect to the least disconnected device if is in the device
```
Comment (boundlesscalm) [inline Changelog.md:20]:
```suggestion
-   No longer auto reselect to the last disconnected device if it is in the device
```

Comment (boundlesscalm) [inline Changelog.md:20]:
```suggestion
-   No longer auto reselect to the last disconnected device if it is in the device
```

Comment (KievDevel) [reply Changelog.md]: Thanks, I did not go through previous versions changelog a lot :)

File: Changelog.md [hunk 2]
```diff
@@ -17,10 +17,10 @@
 
 -   Blocking dialog when disconnecting a device when the Programming dialog is
     open.
--   No longer auto reselect to the least disconnected device if is is in the
-    device list when clicking the `Auto Reconnect` Toggle.
+-   No longer auto reselect to the last disconnected device if it is in the device
```
Comment (boundlesscalm) [inline Changelog.md:20]:
```suggestion
-   No longer autoreconnect to the last disconnected device if it is in the device
```
Sorry I didn't really see this in my last comment

Comment (boundlesscalm) [inline Changelog.md:20]:
```suggestion
-   No longer auto-reconnects to the last disconnected device if it is in the device
```

**Review by kylebonnici — APPROVED** [review]

**Pull request merged**